datafusion/core/src/physical_plan/sorts/cursor.rs

```diff
@@ -110,3 +110,24 @@ impl Ord for SortKeyCursor {
         }
     }
 }
+
```

Comment:
What is the plan for the `Cursor` trait? As in why make at trait that has only a single impl?

Reply:
There will be implementations specialized for the various single column types